index.bs

```diff
@@ -297,8 +297,28 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       then return true.</li>
       <li>Otherwise, for each <var>item</var> in the <a>allowlist</a>:
         <ol>
-          <li>If <var>item</var> is [=same origin-domain=] with
-          <var>origin</var>, then return true.</li>
+          <li>If <var>item</var> is a <a>serialized-origin</a> that's [=same origin-domain=] 
+          with <var>origin</var>, then return true.</li>
+          <li>Else if <var>item</var> is a <a>serialized-origin-with-wildcard-subdomain</a>
+          then:</li>
+          <ol>
+            <li>If <var>origin</var> does not have a host with a <a>registrable domain</a>
+            return false.<li>
+            <li>Set <var>originWithoutWildcard</var> to be <var>item</var> without the "*."
```

Comment:
Maybe use code point substring here? https://infra.spec.whatwg.org/#code-point-substring

Reply:
I dunno, that seems overly complicated, is the updated language okay? I wanted to mirror what https://infra.spec.whatwg.org/#strip-and-collapse-ascii-whitespace did

Reply:
OK, current language seems fine